### PR TITLE
feat: add support for multiple documents format and conversion

### PR DESCRIPTION
**Pull request by HarshitVashisht11**
fixes: #701

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by HarshitVashisht11** [comment]
i have already fill the CLA

File: packages/ui/primitives/document-dropzone.tsx [hunk 1]
```diff
@@ -89,6 +90,8 @@ export const DocumentDropzone = ({
   const { getRootProps, getInputProps } = useDropzone({
     accept: {
       'application/pdf': ['.pdf'],
+      'application/msword': ['.doc'],
```
Comment (Mythie) [inline packages/ui/primitives/document-dropzone.tsx:93]: Seems this isn't included in the processing code above so it'll never work? 🤔

File: apps/web/convertDocumentToPDF.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import jsPDF from 'jspdf';
+import mammoth from 'mammoth';
```
Comment (Mythie) [inline apps/web/convertDocumentToPDF.ts:2]: So there are types for this?

File: apps/web/convertDocumentToPDF.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    // Use jsPDF's html method to add HTML content to the PDF
+    void pdf.html(htmlContent, {
+      callback: (pdf) => {
+        // Output the PDF as a blob
+        const pdfBlob = pdf.output('blob');
+        resolve(pdfBlob);
+      },
+    });
+  });
+}
```
[the remark this answers is not in the document]
Comment (HarshitVashisht11) [reply apps/web/convertDocumentToPDF.ts:26]: The conversion is taking place but the file after converting to pdf dont look like earler file so i am trying to fix this issue

**Comment by HarshitVashisht11** [comment]
i have updated the code you can check the video here 


Uploading Untitled video - Made with Clipchamp (5).mp4…

**Comment by HarshitVashisht11** [comment]
can you suggest any other method which i can try to fix this issue

**Pull request closed without merging**